core/src/main/java/tech/tablesaw/io/string/DataFramePrinter.java

```diff
@@ -29,6 +30,8 @@
  */
 public class DataFramePrinter {
 
+    private final static String TOO_SHORT_COLUMN_MARKER = "?"; 
```

Comment:
Maybe use the Unicode null character? https://www.fileformat.info/info/unicode/char/0000/index.htm

I'm wondering what if someone has a column containing `?`. That might help distinguish

Reply:
I suggested the bomb character, since it stands out and is an unlikely column value. The crucial question is what does the character look like in the console (view)?
BTW, we could also try/catch runtime exceptions and just toString the exception object.